README.md

```diff
@@ -13,7 +13,7 @@ mnesia tables.
 Call `mnesia_eleveldb:register(mnesia_eleveldb)` immediately after
```

Comment:
I see the problem. The code below is correct, it is this line that is wrong. It should simply say mnesia_eleveldb:register(), and then the default table type name "leveldb_copies" will be used.